keyboards/keebio/iris/keymaps/jontonsoup/keymap.c

```diff
@@ -0,0 +1,130 @@
+#include QMK_KEYBOARD_H
```

Comment:
Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

Reply:
Apologies -- I just forked the main repo and followed the directions (maybe I missed this part). Is the idea here that each contributor needs to add a license vs the entire program is not covered by a license? 

Thanks for taking the time to review this!

Reply:
Basically, yup.  Specifically: 
https://www.gnu.org/licenses/gpl-faq.en.html#LicenseCopyOnly
https://www.gnu.org/licenses/gpl-faq.en.html#NoticeInSourceFile

The tl;dr: is that even if the license file exists in the repo, that doesn't cover the individual files if somebody copies them elsewhere.   

So to be explicitly clear on the licensing, it's best to include it in the header for each file.